src/__main__.py

```diff
@@ -38,15 +38,6 @@ async def hi_slash(
 # This must be on the last line, no code will run after this:
 bot.run()
 
-# Note:
-# You should set the interaction URL in the developer portal to point the URL of your server.
-# You may need a domain name for this, as Discord requires SSL. Hikari starts the server on port 8080 by default.
-
-# Tip:
-# For local development, certain editors such as VS Code allow you to create a temporary URL to forward a port through:
-# https://code.visualstudio.com/docs/editor/port-forwarding (Do not forget to set the port publicly accessible!)
-
-# You can also use localhost.run:
-# https://localhost.run/
-
-# This should allow you to test your bot locally without having to deploy it to a server.
+# You can run the bot by executing the following command in the terminal:
+# py -m src on Windows
+# python3 -m src on Linux or MacOS
```

Comment:
This is already mentioned in the README, so I don't think this belongs here.

Reply:
Removed those comments. Let me know if you want to add any other information